zerver/openapi/zulip.yaml

```diff
@@ -15279,7 +15279,7 @@ paths:
 
                               - "recent_topics" - Recent conversations view
                               - "inbox" - Inbox view
-                              - "all_messages" - All messages view
+                              - "global_feed" - Global feed view
```

Comment:
I don't think you can just change this here without also doing a migration to adjust existing values in the database. 

See `web_home_view_options` for a value that needs to be coordinated with this.